### ISSUE TITLE
Final peer review - vap43

### ISSUE THREAD

**Issue by vineetparikh** [adamghx/ORIE4740_Project#10]
This project's premise is to predict, based on medical data available via Kaggle, the likelihood of a person having heart disease generally. As heart disease is a deadly, expensive, and unsettlingly widespread class of diseases, building a predictive model to understand whether a patient might have heart disease, or to understand the factors that tend to contribute to heart disease, would be especially useful. This project attempts to fit an ML model to the Cardiovascular Disease Dataset from Kaggle, after cleaning and preprocessing the data, that can be predictive enough to be useful: after exploring the dataset, they try out different model types (such as Random Forest, SVM, etc.) to see which one works best, and evaluate the results.

As the dataset itself represents a binary classification task, the model evaluation this team used was simply to evaluate exact accuracy (i.e. l1 error), and they had found that the model that worked best on their training data (Random Forest, a black-box model) was also potentially overfitting to the training data. The group was able to reduce this overfitting via cross-validation, dropping some features, and tuning hyperparameters within the Random Forest model (using grid search) to get a fairly solid level of testing accuracy (with a training accuracy of 74%, the model had a test accuracy of 73.5%): in fact, this model could potentially top the Kaggle leaderboard for this particular problem!

This report is really good! I loved the extensive discussions on data exploration and model selection, the highly specified way in which hyperparameter tuning and cross-validation were employed to reduce overfitting, and a solid understanding of how hard this problem really is as exemplified in both the problem statement and in the rest of the document. I do think there are a few additional points of feedback to give (mostly as extensions). Firstly, while the more complex models on scikit-learn are definitely not super interpretable, it'd be neat to see how this use-case could benefit from inserting complex models that are more interpretable by design (such as interpretml from MSR). Secondly, while there was information given on how to split the data between training and testing, I'm not quite sure why an 85-15 split was used, and it'd be cool to see if this type of model was compared with similar parameters as the Kaggle competition's top models (i.e. using a data split similar to the neural network that has the current highest performance, to enable an apples-to-apples comparison). Finally, I'm not quite sure if I'd agree with how the report considers the fairness of their model: as certain areas could have less medical data available, even for more widespread conditions like heart disease, it might be possible that medical data corresponding to those areas could be scarce and could lead to models not being equipped to fairly handle this. As the model selected was definitely a black-box model that is used to handle healthcare, the lack of interpretability could make it hard for individual doctors to course-correct.
